docs/features/scripting.rst

```diff
@@ -94,6 +94,24 @@ where:
 * ``command`` and ``args`` are entered exactly like they would be entered by
   a user of your application.
 
+Advanced Support
+~~~~~~~~~~~~~~~~
+
+When implementing a command, setting ``self.__last_result`` allows for application-specific
```

Comment:
I think this is incorrect and should be `self.last_result` (applies below as well)